examples/launcher/src/components/Engine.jsx

```diff
@@ -1,8 +1,19 @@
 import React from 'react';
+import Resizable from 're-resizable';
 import Dom from './Dom';
 import Console from './Console';
 import '../css/engine.css';
 
+const postViewportMessage = () => {
```

Comment:
This is a nitpick, but in case it's not clear, we generally prefix "private" methods with `_` in the codebase.

"Private" means "not exported out of this scope". In lieu of a true language feature for it, it makes it easier to tell at a glance the intent of the code.

Reply:
e447ac0